package.json

```diff
@@ -31,6 +31,7 @@
     "winston": "^3.1.0"
   },
   "devDependencies": {
+    "jsdom": "^13.0.0",
```

Comment:
I don't see it used...

Reply:
it is used in the tests heavily…and in the current version to provide the getComputedStyle function for dom.js…

Reply:
yes. but not in this PR...so I don't understand why it is only added now. nevermind.